CHANGELOG.md

```diff
@@ -5,6 +5,126 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.10.0] - unreleased
+
+### Added
+
+- Crate features
+  - `serde`: support for serialization/deserialization of `Variant` and core types (743)
```

Comment:
This should probably be:
```suggestion
  - `serde`: support for serialization/deserialization of `VariantDispatch` and core types (743)
```

Hopefully serialization of `Variant` itself can be added by `10.1`... Maybe you could also add something like

> (*serde support for `Variant` comming soon™*)

Reply:
Thanks, fixed! 

Haha 😄 I don't want to promise too much yet, still need to get 0.10 done... and then there's always Godot 4 and the huge safety topic 😬